### DISCUSSION TITLE
apk to aab

### DISCUSSION THREAD

**Discussion by orcuncun** [airsdk/Adobe-Runtime-Support#1213] [Guides & Tutorials]
Is it possible to convert .apk package to .aab package?

**Comment by ajwfrost** [comment]
Not as far as I'm aware... some of the elements in an AAB are in completely different formats so I don't know how easy it would be to take an APK and convert it into a (limited) AAB file.

**Reply by orcuncun** [reply]
a program that allows me to keep the source code only works with apk packages.

In that case, should I wait for that program to release a version for .aab packages?
if not possible?

**Reply by ajwfrost** [reply]
Not sure I fully understand but yes it sounds like it, if it's a third party program that's generating an APK then that program would need to instead output an AAB package if it's going to be something that's then uploaded to the Play Store.
